src/routes/api/users.js

```diff
@@ -247,8 +252,99 @@ router.post('/auth/forgotPassword', catchErrors(users.forgotPassword));
  *      '404':
  *        description: email not found
  *      '200':
- *        description: succesfull sent reset link
+ *        description: successfully sent reset link
  */
 router.patch('/auth/resetPassword', validation, decodeQueryToken, catchErrors(users.resetPassword));
 
+/**
+ * @swagger
+ *
+ * /user/update-profile:
+ *  patch:
+ *    tags:
+ *      - Users
+ *    summary: Users profile settings
+ *    description: User gets a user profile upon successful registration to Barefoot Nomad
+ *      and is able to update/edit
+ *    operationId: CreateUserProfile
+ *    produces:
+ *      - application/json
+ *    parameters:
+ *      - in: body
+ *        name: user
+ *        schema:
+ *          type: object
+ *          properties:
+ *            firstName:
+ *              type: string
+ *              example: john
+ *            lastName:
+ *              type: string
+ *              example: doe
+ *            password:
+ *              type: string
+ *              example: password
+ *            gender:
+ *              type: string
+ *              example: male
+ *            lineManager:
+ *              type: string
+ *              example: Williams Doe
+ *            preferredCurrency:
+ *              type: string
+ *              example: "$"
+ *            preferredLanguage:
+ *              type: string
+ *              example: English
+ *            department:
+ *              type: string
+ *              example: IT
+ *            birthDate:
+ *              type: string
+ *              example: 14/10/1990
+ *            residenceAddress:
+ *              type: string
+ *              example: 14, Jeremiah Ugwu, Lekki, Lagos
+ *            phoneNumber:
+ *              type: integer
+ *              example: 08012345678
+ *    responses:
+ *      '201':
+ *        description: successful operation
+ *      '404':
+ *        description: Unsuccessful User not found
+ *      '500':
+ *        description: Internal Server error
+ * */
+router.patch('/user/update-profile', verifyUser, validation, catchErrors(updateUserInfo));
+
+/**
+ * @swagger
+ *
+ * /user/:userId/get-profile:
+ *  get:
+ *    tags:
+ *      - Users
+ *    summary: Users profile settings
+ *    description: Barefoot Nomad User is able to access their Barefoot Nomad profile
+ *    operationId: GetUserProfile
+ *    produces:
+ *      - application/json
+ *    parameters:
+ *      - name: userId
+ *        in: path
+ *        description: ID of user to return a profile
+ *        required: true
+ *        type: integer
+ *        format: int64
+ *    responses:
+ *      '201':
+ *        description: successful operation
+ *      '404':
+ *        description: Unsuccessful User not found
+ *      '500':
+ *        description: Internal Server error
+ * */
+router.get('/user/:userId/get-profile', verifyUser, catchErrors(getUserProfile));
```

Comment:
I don't think `/get-profile` is necessary here, `/user/:userId` should do.

Reply:
Ok thanks, let me change the route that way.